projects/plugins/boost/app/assets/src/js/modules/image-size-analysis/RecommendationsMeta.svelte

```diff
@@ -67,6 +68,15 @@
 		}
 	}
 
+	function handleAnalyzeClick() {
+		const $event_name =
+			$isaSummary.status === ISAStatus.Completed
+				? 'clicked_restart_isa_on_summary_page'
+				: 'clicked_start_isa_on_summary_page';
+		recordBoostEvent( $event_name, {} );
+		return onStartAnalysis();
```

Comment:
Renamed `onStartAnalysis` to just `startAnalysis` in 2ae83afece. I don't mind which convention we use; `handleX` or `onX`, but `on...` implies it's an event handler, and it no longer is. It's just being called from an event handler. :)